### PR TITLE
Security fix for addon-resizer v1.8.16

### PR DESCRIPTION
**Pull request by ShaunakJoshi1407**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: addon-resizer/Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ ARCH ?= amd64
 export DOCKER_CLI_EXPERIMENTAL=enabled
 
 GOARM=7
-GOLANG_VERSION = 1.16
+GOLANG_VERSION = 1.17
 REGISTRY = gcr.io/k8s-staging-autoscaling
```
Comment (cruizen) [inline addon-resizer/Makefile:26]: Now that we have forked from the upstream repo, we should change the registry and tag.

**Review by cruizen — CHANGES_REQUESTED** [review]
We need to revisit, change where the Docker image is published.

**Comment by ShaunakJoshi1407** [comment]
__

**Comment by cruizen** [comment]
> __

Why did we close the PR? Can we add any detail?

**Comment by ShaunakJoshi1407** [comment]
> > __
> 
> Why did we close the PR? Can we add any detail?

Closed this PR out and raised a new PR here: https://github.com/platform9/cluster-autoscaler/pull/2 that includes a target for pushing the image to Platform9 Dockerhub.

**Pull request closed without merging**